actionview/lib/action_view/helpers/url_helper.rb

```diff
@@ -202,10 +182,28 @@ def _filtered_referrer # :nodoc:
       # * <tt>:disable_with</tt> - Value of this parameter will be used as the
       #   name for a disabled version of the link. This feature is provided by
       #   the unobtrusive JavaScript driver.
+      # * <tt>remote: true</tt> - This will allow @rails/ujs
+      #   to make an Ajax request to the URL in question instead of following
+      #   the link. The drivers each provide mechanisms for listening for the
+      #   completion of the Ajax request and performing JavaScript operations once
+      #   they're complete
+      # * <tt>method: symbol of HTTP verb</tt> - This modifier will dynamically
+      #   create an HTML form and immediately submit the form for processing using
+      #   the HTTP verb specified. Useful for having links perform a POST operation
+      #   in dangerous actions like deleting a record (which search bots can follow
+      #   while spidering your site). Supported verbs are <tt>:post</tt>, <tt>:delete</tt>, <tt>:patch</tt>, and <tt>:put</tt>.
+      #   Note that if the user has JavaScript disabled, the request will fall back
+      #   to using GET. If <tt>href: '#'</tt> is used and the user has JavaScript
+      #   disabled clicking the link will have no effect. If you are relying on the
+      #   POST behavior, you should check for it in your controller's action by using
+      #   the request object's methods for <tt>post?</tt>, <tt>delete?</tt>, <tt>patch?</tt>, or <tt>put?</tt>.
```

Comment:
Unlike `:confirm` and `:disable_with`, these options can be specified as top-level options (i.e. not as part of `data: {}`).  Doing so activates some (minor) [additional logic](https://github.com/rails/rails/blob/98b8b0dd707373f45ccb8b0f8ab6fa653ce99ddc/actionview/lib/action_view/helpers/url_helper.rb#L748-L757).  Perhaps it would be good to separate these options?  Maybe add 

```ruby
# @rails/ujs also integrated with the following +:data+ options:
``` 

below `:method` and `:remote`, and then move `:confirm` and `:disable_with` below that.  What do you think?

Reply:
Ahh yep good call.

Reply:
I updated it as suggested in both places.